index.ts

```diff
@@ -1239,6 +1385,7 @@ export const translations = {
         "TE Snelheid",
         "EC rychlost",
         "Velocità ER",
+        "",
```

Comment:
teSpeed - Could you provide some article/wiki with definition what it is? It will help with translation.

Reply:
Total energy speed is your total energy (kinetic + potential) in relation to your task.
Example:
Deviating from your planned flight path results in a faster loss of TE-Speed but when it happens because there is a good climb/thermal, TE-Speed can increase while thermaling.